src/liboslexec/batched_backendllvm.cpp

```diff
@@ -667,7 +667,12 @@ BatchedBackendLLVM::llvm_get_pointer(const Symbol& sym, int deriv,
             arrayindex = ll.op_add(arrayindex, ll.constant(d));
         else
             arrayindex = ll.constant(d);
-        result = ll.GEP(result, arrayindex);
+
```

Comment:
Add comment here:
`// Array's will not be forced_llvm_bool, so no need to check`